Okta.Xamarin/Okta.Xamarin/OidcClient.cs

```diff
@@ -28,6 +28,11 @@ public abstract class OidcClient : IOidcClient
         /// </summary>
         public event EventHandler<RequestExceptionEventArgs> RequestException;
 
+        /// <summary>
+        /// The event that is raised when the exchange of code for tokens throws an exception.
+        /// </summary>
+        public event EventHandler<AuthCodeTokenExchangeExceptionEventArgs> AuthCodeTokenExchangeException;
```

Comment:
Since this is an event, should we add the `On` prefix for clarity (`OnAuthCodeTokenExchangeException`)?

Reply:
The `On` prefix is used for methods that take responsibility for invoking the event, the event itself wouldn't have the prefix.  I can refactor in the future if it becomes necessary to allow an inheritor of OidcClient to invoke the `AuthCodeTokenExchangeException` event.

Reply:
Actually, looking closer, I did include the OnAuthCodeTokenExchangeException protected method intended to invoke the event.

Reply:
Then, maybe this should use the event suffix. Otherwise, when reading the code this can be interpreted as an exception instead.

Reply:
I can see that.  I could change the name back to `AuthCodeTokenExchangeFailed`.

Reply:
Or better yet, I don't want to assume that the exchange failed, I could change it to `AuthCodeTokenExchangeExceptionThrown`.